src/v3/src/OktaSignIn/index.ts

```diff
@@ -120,8 +120,7 @@ export default class OktaSignIn {
       // eslint-disable-next-line no-underscore-dangle
       const userAgent = this.authClient._oktaUserAgent;
       if (userAgent) {
-        userAgent.addEnvironment('okta-signin-widget-next');
-        userAgent.addEnvironment(OKTA_SIW_COMMIT_HASH);
+        userAgent.addEnvironment(`okta-signin-widget-next-${this.version}`);
```

Comment:
Wondering should we make it something like 
`userAgent.addEnvironment(`okta-signin-widget-g3-${this.version}`);` or stick with next? Either works for me, just curious if we were officially trying to move from the `next` moniker.

Reply:
Oh you already mentioned this in the slack thread. Sorry for repeating :(

Reply:
No that's a good idea, I'll update to reflect a generation value